charts/plot-navigator/values.yaml

```diff
@@ -0,0 +1,19 @@
+hostname: ""
+# Include leading forward slash in basePath
+basePath: "/plot-navigator"
+image:
+  repository: lsstdm/pipetask-plot-navigator
+  tag: ""
+
+# This is the path to a butler.yaml file.
+# e.g. s3://butler-us-central1-panda-dev/dc2/butler-external.yaml
+butler_uri: ""
+
+# override in phalanx
+butler_secret_path: ""
+
+ingress:
+  enabled: true
+  host: ""
+  path: "/plot-navigator"
+  pathType: ImplementationSpecific
```

Comment:
Consider using [helm-docs](https://github.com/norwoodj/helm-docs) to document each of the meaning of each of these values, and then generating a README.md with helm-docs. 

For example, here's Gafaelfawr's [values.yaml](https://github.com/lsst-sqre/charts/blob/master/charts/gafaelfawr/values.yaml) and the resulting generated [README.md](https://github.com/lsst-sqre/charts/blob/master/charts/gafaelfawr/README.md).